src/MainHeader.css

```diff
@@ -1,6 +1,9 @@
 .title-container {
-  background-color: lightcyan;
+  background-color: white;
```

Comment:
white is the default, so this isn't needed. the next commented line can go away too